README.md

```diff
@@ -16,8 +17,10 @@ New package dependencies for the project can be added using the `poetry add <pac
 
 ## Setting up the Docker Environment
 
-Before starting up the container, make sure to run the command `python manage.py makemigrations` if you have made any changes to the database models. 
+Before starting up the container, make sure to run the command `python manage.py makemigrations` if you have made any changes to the database models. This should be done inside of the `evergreen` directory with the poetry virtual environment activated. 
```

Comment:
`makemigrations` is already run on container creation, and the container should be recreated on any change that would trigger requiring a migration, so this is unnecessary